test/e2e/e2e_test.go

```diff
@@ -31,16 +31,21 @@ var (
 // TestScenarios runs all the e2e tests. Any new tests need to be added to this
 // list in order for them to run.
 func TestScenarios(t *testing.T) {
+	// These scenarios run as part of the PR presubmit job
 	tests := map[string]func(t *testing.T){
```

Comment:
Instead of commenting them out, we could also select through `test.run`, that way we can still run these tests e.G. in a periodic

Reply:
oh man, I forgot about `test.run`.  Would should just use that and tag our tests with something like `[Base]`.